Remove sys. Use util instead.

jaumesala opened the pull request:
Remove warning: (node) sys is deprecated. Use util instead.
https://github.com/nodejs/node-v0.x-archive/wiki/deprecation

jaumesala committed:
Remove sys. Use util instead.

Remove warning: (node) sys is deprecated. Use util instead.
https://github.com/nodejs/node-v0.x-archive/wiki/deprecation

tmcw commented on csv2geojson:
```diff
@@ -2,7 +2,7 @@
 var optimist = require('optimist'),
     fs = require('fs'),
-    sys = require('sys'),
+    sys = require('util'),
```
please rename the variable as well, and references to it, to avoid the technical debt of a misnamed require

jaumesala replied:
Sorry, I went too fast to commit. I dug up a little bit and it looks like it is not used anymore.

jaumesala committed:
Removed sys/util. Module not used

I've just realised this module is not used.
https://github.com/mapbox/csv2geojson/commit/8980ac895f38a3073ac62163519945d27b3ff25d

tmcw commented:
👍 Thanks!
